packages/@n8n/permissions/src/constants.ee.ts

```diff
@@ -23,4 +23,5 @@ export const RESOURCES = {
 	workersView: ['manage'] as const,
 	workflow: ['share', 'execute', 'move', ...DEFAULT_OPERATIONS] as const,
 	folder: [...DEFAULT_OPERATIONS] as const,
+	insights: [...DEFAULT_OPERATIONS] as const,
```

Comment:
Do we need `DEFAULT_OPERATIONS` here or should we scope it to `list` and `read` only? We're not going some of them like `delete` are we?

Reply:
good point @MarcL. Maybe just a `list` one then, because we won't read only one of them alone, no ? what do you think :thinking: ?

Reply:
Sounds good. We could always expand on it in a future PR.